webui/tests/test-cluster/master.yaml

```diff
@@ -17,4 +17,4 @@ db:
   password: postgres
 
 root: build
-http_port: 8081
+port: 8081
```

Comment:
this should already be in through https://github.com/determined-ai/determined/pull/1366.

Reply:
yep will rebase